doc/modules/cross_validation.rst

```diff
@@ -558,6 +558,35 @@ Here is a visualization of the cross-validation behavior.
 :class:`RepeatedStratifiedKFold` can be used to repeat Stratified K-Fold n times
 with different randomization in each repetition.
 
+Binned Stratified k-fold
```

Comment:
I would add this section below so that `StratifiedKFold` and `StratifiedShuffleSplit` are still together